Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -1772,6 +1772,10 @@ InternalBootPickerAnimateIntro (
       );
   }
 
+  if (CurrentTime == 0) {
+    PrevSine = 0;
+  }
+
```

Comment:
This is a hack, but having a static PrevSine value is (arguably?) a hack anyway.